### PR TITLE
OCPBUGS-46497:updating monitored accelerator list

### PR DESCRIPTION
**Pull request by yevgeny-shnaidman**

### CODE REVIEW

**Comment by simonpasquier** [comment]
@yevgeny-shnaidman you'll need to create an OCPBUGS ticket (assigned to the Monitoring component) and link it to the PR.
/label backport-risk-assessed

File: collector/accelerators_test.go [hunk 1]
```diff
@@ -39,9 +39,8 @@ func (c testAcceleratorCollector) Describe(ch chan<- *prometheus.Desc) {
 func TestAccelerator(t *testing.T) {
 	testcase := `# HELP node_accelerator_card_info Accelerator card info including vendor, model and pci id (address)
 	# TYPE node_accelerator_card_info counter
-	node_accelerator_card_info{id="0000:00:02.0",model="A100",vendor="NVIDIA"} 1
-	node_accelerator_card_info{id="0000:00:09.0",model="A100",vendor="NVIDIA"} 1
-	node_accelerator_card_info{id="0000:00:1f.5",model="RTX_4090",vendor="NVIDIA"} 1
```
Comment (simonpasquier) [inline collector/accelerators_test.go:44]: not sure why this line needs to be removed?

Comment (yevgeny-shnaidman) [reply collector/accelerators_test.go:44]: This accelerator is no longer monitored, so the metric won't be created

**Comment by simonpasquier** [comment]
this PR needs #159 to be merged to remove the `jira/invalid-bug` label.

**Pull request merged**